internal/translator/test/rules.go

```diff
@@ -11,37 +11,97 @@ import (
 // TestRule contains a rule that match with a `det,noun` sentence. It will generate
 // the inverse `noun,det` with the same words
 var TestRule = `
-{
-	"source_language": "es",
-	"target_language": "kq",
-	"pattern": "DET,NOUN",
-	"details": [
-		{
-			"tag": "DET",
-			"type": "",
-			"properties": {}
-		},
-		{
-			"tag": "NOUN",
-			"type": "",
-			"properties": {}
-		}
-	],
-	"output": [
-		{
-			"type": "direct-translation",
-			"value": "{{ .Word2.Lemma }}"
-		},
-		{
-			"type": "literal",
-			"value": " "
-		},
-		{
-			"type": "literal",
-			"value": "{{ .Word1.Lemma }}"
-		}
-	]
-}
+[
+	{
+		"source_language": "es",
+		"target_language": "kq",
+		"pattern": "DET,NOUN",
+		"details": [
+			{
+				"tag": "DET",
+				"type": "A",
```

Comment:
It will be like happened with the example I modified, It is going to return the last match in the case doesn't have any Type or the rule that have the more Type matches, but good observation, thanks, if the rule doesn't have any type, it is better don't go to evaluate the rule by type.
That will be done in the project